Common_Files/Physics.ipynb

```diff
@@ -10,19 +10,53 @@
+    "\n",
+    "$$FD=f (d,μ,ρ,V)$$\n",
+    "\n",
+    "where d is the dimension of the rectangle, μ and ρ are the viscosity and density of the fluid. As there are 5 variables and 3 reference dimensions (i.e., Mass, Length, Time), two non-dimensional groups (i.e. Π groups) are needed to fully define the flow field. Performing dimensional analysis result in two following non-dimensional groups:\n",
+    "\n",
+    "$$Π1=ρVdμΠ1=ρVdμ$$\n",
+    "\n",
+    "$$Π2=FDd2ρV2Π2=FDd2ρV2$$\n",
+    "\n",
+    "Therefore, as shown previously using control volume analysis, it can be concluded the drag force on the rectangle is a function of velocity field through an unknown function ϕ as follows:\n",
+    "\n",
+    "$$FDd2ρV2=ϕ(ρVdμ)FDd2ρV2=ϕ(ρVdμ)$$\n",
+    "\n",
+    "The left hand side of this equation is the non-dimensional drag force and referred to as the drag force coefficient. The right hand side of this equation is the ratio of inertial forces to the viscous forces within the flow, which is known as Reynolds number. This outcome of the dimensional analysis confirms that the drag force on the rectangle in a uniform flow is a direct function of Reynolds number. Therefore, majority of scientists investigating on this flow field look at the variation of these two variables to find the function ϕ.\n",
+    "\n",
+    "$$CD=ϕ(Re)$$\n",
+    "\n",
+    "In this equation CD, the total coefficient of the drag force due to pressure and viscous forces within the flow acting on the rectangle's surface in the stream wise direction. This coefficient is the normalized drag force component with the available momentum in the undisturbed incoming flow that could be exerted on the rectangle. The value of this momentum is calculated using ρ as the reference density of the flow, which is the density of the flow. V as the undisturbed flow velocity in the inlet and d is the diameter of the rectangle surface. In some formulation and books the term d2 is replaced with A as the projected area of the rectangle. For a two-dimensional rectangle the projected area is equal to the diameter of the rectangle. This is the area where pressure and viscous forces are acting on the rectangle in the stream wise direction. It should be reminded that the pressure and viscous forces in the perpendicular direction to the flow direction cancel each other. Hence, the forces and areas that they act on does not have any contribution to the drag force and coefficient.\n",
```

Comment:
is "d" diameter of the rectangle or width of it that is seeing the incoming flow?

Please confirm/correct it.